x-pack/plugins/cases/public/components/case_view/components/case_view_activity.tsx

```diff
@@ -290,10 +295,10 @@ export const CaseViewActivity = ({
             />
           ) : null}
           <CustomFields
-            isLoading={isLoading && loadingKey === 'customFields'}
+            isLoading={(isLoading && loadingKey === 'customFields') || isUpdatingCustomField}
```

Comment:
I would suggest mocking this in `case_view_activity.test.tsx` to guarantee `isUpdatingCustomField` is `false` and these changes don't introduce flakyness in the tests.

Kinda like what we do [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/public/components/case_view/components/case_view_activity.test.tsx#L169) with `isLoading`.